tools/make_docs

```diff
@@ -1,38 +1,39 @@
 #!/bin/sh
 
```

Comment:
Add

```shopt -s xpg_echo```

to have Bash expand the escape sequences in echo. I tried that with dash (the default shell in Debian/Ubuntu) and everything still works.